code/src/App.js

```diff
@@ -1,9 +1,42 @@
-import React from 'react'
+import React, { useState }  from 'react'
+import { NameQuestion }  from './Components/NameQuestion.js';
+import { NoseQuestion }  from './Components/NoseQuestion.js';
+import { EyeQuestion }  from './Components/EyeQuestion.js';
+import { SmileQuestion }  from './Components/SmileQuestion.js';
+import { MustacheQuestion }  from './Components/MustacheQuestion.js';
+import { Heading }  from './Components/Heading.js';
+import { SubmitQuestions }  from './Components/SubmitQuestions.js';
+import { Respons }  from './Components/Respons.js';
```

Comment:
Remember to separate imports Components that are from node_modules, and your's with one empty line.